### DISCUSSION TITLE
new exploits

### DISCUSSION THREAD

**Discussion by SN-Reality** [3kh0/ext-remover#1391] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
is there anyone looking for any new exploits for 130 or would I just have to downgrade to use any

**Comment by sites1235** [comment]
You can try BR1CK

**Reply by SN-Reality** [reply]
could u send me a pdf or ss of the instructions because my school has the website blocked

**Reply by S-PScripts** [reply]
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Unenrollment%20-%20BR1CK

**Reply by SlitheringLizard1** [reply]
br1ck sound too risky and it would wipe wifi data

**Reply by S-PScripts** [reply]
> it would wipe wifi data

then get the wifi passwords ?

**Reply by prjamming** [reply]
You can follow this guide to get the wifi passwords from your chromebook.
https://www.youtube.com/watch?v=yXwORaJh0PQ

**Reply by SlitheringLizard1** [reply]
ok so about that

I recently went homeschool and I know what my wifi password is
im good

**Comment by ghost** [comment]
U can try my exploit

**Reply by akabutnicer** [reply]
patched on v131

**Reply by akabutnicer** [reply]
> will update on January 24 2025

you should have found the new method before you posted this 🤦

**Reply by jazzymakesstuff** [reply]
lmao

**Reply by DocOcc** [reply]
> > will update on January 24 2025
> 
> you should have found the new method before you posted this 🤦

What new method?  Had caubDNS working perfectly on version 115 for like 2 years before some dipshit power washed it.  At least I can use youtube music again...

**Reply by ghost** [reply]
no didnt found i tried it on an unused 108 chromebook (hana)
